package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "edx-ora2",
-  "version": "2.8.8",
+  "version": "2.8.9",
   "repository": "https://github.com/edx/edx-ora2.git",
   "dependencies": {
     "backbone": "~1.2.3",
     "backgrid": "~0.3.8",
     "edx-ui-toolkit": "1.5.3",
-    "moment": "^2.15.1",
```

Comment:
Did you do this or was it automatic? What is it?

Reply:
Automatic.  Result from a  `make javascript sass` command